test/companies/companies_test.exs

```diff
@@ -13,69 +13,10 @@ defmodule Companies.CompaniesTest do
     end
   end
 
-  # @test "filters companies by industry" do
```

Comment:
Just a nitpicl but I removed all of this because we have it in source control and I don't love having huge blocks of commented code living for long periods in any codebase.